lib/voodoo_sms.rb

```diff
@@ -14,6 +14,14 @@ class RequiredParameter < StandardError; end
     class InvalidParameterFormat < StandardError; end
   end
 
+  class Response
+    attr_reader :reference_id
+
+    def initialize(reference_id)
+      @reference_id = reference_id
```

Comment:
:+1: 

It might be worth expanding some test cases to see what happens if you change the response from VoodooSMS, I suspect `Response.reference_id` will return nil in this case but is this something we want? (I don't know)